deployments/kubernetes/reloader.yaml

```diff
@@ -41,6 +41,9 @@ spec:
         image: "stakater/reloader:v0.0.31"
         imagePullPolicy: IfNotPresent
         name: reloader
+        args:
+          - "--resources-to-watch=secrets"
```

Comment:
@henryaj kindly rename this arg to `--resources-to-ignore` in all files because in `/template/deployment` file you have used `--resources-to-ignore`.

Reply:
Good catch!